pkg/apis/seedmanagement/types_managedseed.go

```diff
@@ -0,0 +1,182 @@
+	// Specification of the desired behavior of the Seed.
+	Spec gardencorev1beta1.SeedSpec
+}
+
+// Gardenlet specifies gardenlet deployment parameters and the GardenletConfiguration used to configure gardenlet.
+type Gardenlet struct {
+	// Deployment specifies certain gardenlet deployment parameters, such as the number of replicas,
+	// the image, which bootstrap mechanism to use (bootstrap token / service account), etc.
+	Deployment *GardenletDeployment
+	// Config is the GardenletConfiguration used to configure gardenlet.
+	Config *gardenletconfigv1alpha1.GardenletConfiguration
+	// Bootstrap is the mechanism that should be used for bootstrapping gardenlet connection to the Garden cluster. One of ServiceAccount, Token.
+	// If specified, a service account or a bootstrap token will be created in the garden cluster and used to compute the bootstrap kubeconfig.
+	// If not specified, the gardenClientConnection.kubeconfig field will be used to connect to the Garden cluster.
+	GardenConnectionBootstrap *GardenConnectionBootstrap
+	// SeedConnection is the mechanism for gardenlet connection to the Seed cluster. Must equal ServiceAccount if specified.
+	// If not specified, the seedClientConnection.kubeconfig field will be used to connect to the Seed cluster.
+	SeedConnection *SeedConnection
+	// MergeParentConfig specifies whether the deployment parameters and GardenletConfiguration of the parent gardenlet
+	// should be merged with the specified deployment parameters and GardenletConfiguration. Defaults to false.
+	MergeParentConfig bool
+}
+
+// GardenletDeployment specifies certain gardenlet deployment parameters, such as the number of replicas,
+// the image, which bootstrap mechanism to use (bootstrap token / service account), etc.
+type GardenletDeployment struct {
+	// ReplicaCount is the number of gardenlet replicas. Defaults to 1.
+	ReplicaCount *int32
+	// RevisionHistoryLimit is the number of old gardenlet ReplicaSets to retain to allow rollback. Defaults to 10.
+	RevisionHistoryLimit *int32
+	// ServiceAccountName is the name of the ServiceAccount to use to run gardenlet pods.
+	ServiceAccountName string
+	// Image is the gardenlet container image.
+	Image *Image
+	// Resources are the compute resources required by the gardenlet container.
+	Resources *corev1.ResourceRequirements
+	// PodLabels are the labels on gardenlet pods.
+	PodLabels map[string]string
+	// PodAnnotations are the annotations on gardenlet pods.
+	PodAnnotations map[string]string
+	// AdditionalVolumes is the list of additional volumes that should be mounted by gardenlet containers.
+	AdditionalVolumes []corev1.Volume
+	// AdditionalVolumeMounts is the list of additional pod volumes to mount into the gardenlet container's filesystem.
+	AdditionalVolumeMounts []corev1.VolumeMount
+	// Env is the list of environment variables to set in the gardenlet container.
+	Env []corev1.EnvVar
+	// VPA specifies whether to enable VPA for gardenlet. Defaults to false.
```

Comment:
I think, the default should be `true` for `ManagedSeed`s.
```suggestion
	// VPA specifies whether to enable VPA for gardenlet. Defaults to true.
```

Reply:
OK